README.md

```diff
@@ -80,46 +81,43 @@ Unit Testing
 If you are able to run docker natively, then:
  - make it-now
 
```

Comment:
```suggestion
 - make it-now

```
Since there's only one option remaining, the "If..." statement is less helpful. Could also be shortened to simply "make" since "it-now" is the default target.

Reply:
Fixed by rewriting a bit. :)